FrejaEidClient/src/main/java/com/verisec/frejaeid/client/enums/AttributeToReturn.java

```diff
@@ -91,7 +93,14 @@
      * Current registration level of the user.
      * Corresponds to one of the values of {@linkplain RegistrationLevel}.
      */
-    REGISTRATION_LEVEL("REGISTRATION_LEVEL");
+    REGISTRATION_LEVEL("REGISTRATION_LEVEL"),
```

Comment:
Can you please update description for EMAIL, that primary will be returned unless RP has preferred email domain?